numba/core/byteflow.py

```diff
@@ -1088,6 +1182,35 @@ def _dup_topx(self, state, inst, count):
         for val in duped:
             state.push(val)
 
+    if PYVERSION in ((3, 12), ):
+        def op_CALL_INTRINSIC_1(self, state, inst):
+            # See https://github.com/python/cpython/blob/v3.12.0rc2/Include/
+            # internal/pycore_intrinsics.h#L3-L17C36
+            if inst.arg == 3:  # INTRINSIC_STOPITERATION_ERROR
```

Comment:
Would it be clearer to write this branching on the value of `inst.arg` as `if..elif...elif...else raise NotImplementedError(...)`? (and other similar uses).

Reply:
Think this is fixed in: 5071096b731c1108ad5a906a71e77783a6924f27